prometheus/k8s/service_monitor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
How will this work with non-DSE?

Reply:
As long as the services to be monitored have labels it will perform the matching and configure Prometheus to scrape the endpoints.

Reply:
Our operator has a number of labels included on the appropriate resources to integrate here.